doc/spec/def.proto

```diff
@@ -100,6 +103,8 @@ message ContextInfo {
 
 message SenderKeyDistributionMessage {
     optional string groupId = 1;
+     optional string participant = 2;
```

Comment:
❓ `participant` and `axolotlSenderKeyDistributionMessage` share the same tag ID (`2`)

Which should take priority?